Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm

```diff
@@ -92,6 +92,41 @@ static dispatch_queue_t globaVideoCaptureSerialQueue()
     return globalQueue;
 }
 
+static std::optional<AVCaptureWhiteBalanceMode> whiteBalanceModeFromMeteringMode(MeteringMode mode)
```

Comment:
Why optional, can we just return a default value, say AVCaptureWhiteBalanceModeAutoWhiteBalance

Reply:
True, changed